packages/buidler-core/test/internal/sentry/anonymizer.ts

```diff
@@ -118,4 +118,83 @@ describe("Anonymizer", () => {
       assert.isTrue(anonymizationResult.anonymizeContent);
     });
   });
+
```

Comment:
I think this tests won't pass on windows

Reply:
They seem to have passed

Reply:
I'm confused. Path.sep shouldn't match slash there. Right?

Reply:
Nevermind, you are constructing test paths using things like __filename